app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,13 @@
+from players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str, musical_instrument: str, bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return self._bow_level
+
+    def player_info(self) -> str:
+        return f"Elf Ranger: {self.nickname}"
```

Comment:
The method `player_info` should include all relevant attributes of the ElfRanger. Consider including `self._bow_level` in the returned string for more comprehensive player information.